Modules/Core/Common/test/itkImageRegionIteratorTest.cxx

```diff
@@ -93,9 +93,6 @@ itkImageRegionIteratorTest(int, char *[])
   (*o3)[regionEndIndex3D] = (*o3)[regionStartIndex3D];
   TestConstPixelAccess(*o3, *o3);
 
-
-  itk::ImageIterator<itk::Image<itk::Vector<unsigned short, 5>, 3>> standardIt(o3, region);
```

Comment:
Nice catch, Jon, how dit you find it? Looks like `standardIt` has been there unused for a long time already!

Looking back in May 28, 2008:
https://github.com/InsightSoftwareConsortium/ITK/blob/f2c4bb4042626eb94449662ea758691b3a411b8a/Testing/Code/Common/itkImageRegionIteratorTest.cxx#L100

Reply:
I found it reading the code for other reasons.